src/main.ts

```diff
@@ -82,11 +82,7 @@ class TemplateEntityRow extends LitElement {
   }
 
   render() {
-    const base = this.hass.states[this.config.entity];
-    const entity = (base && JSON.parse(JSON.stringify(base))) || {
-      entity_id: "light.",
-      attributes: { icon: "no:icon" },
-    };
+    const entity = this.hass.states[this.config.entity];
```

Comment:
@michalolborski I don't think you need this change? This seems to cause issues when leaving the entity blank and works fine without this change